### PR TITLE
Handle Cancelled MHR for Staff and QS 

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#20567

*Description of changes:*
- Handle Cancelled MHRs
- Add modal for QS when trying to add Cancelled MHR to the table
- Unit tests
 
Screenshot 2024-05-31 at 13 10 10

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1907-skxdqt7v.web.app

File: ppr-ui/src/components/common/RegistrationsWrapper.vue [hunk 1]
```diff
@@ -329,7 +337,7 @@ export default defineComponent({
       // Getters
       getRegTableBaseRegs, getRegTableDraftsBaseReg, isMhrRegistration, isMhrManufacturerRegistration,
       getRegTableTotalRowCount, getStateModel, getRegTableDraftsChildReg, hasMorePages, getRegTableNewItem,
-      getRegTableSortOptions, getRegTableSortPage, getUserSettings, getMhRegTableBaseRegs, isRoleStaffReg
+      getRegTableSortOptions, getRegTableSortPage, getUserSettings, getMhRegTableBaseRegs, isRoleStaffReg, isRoleQualifiedSupplier
```
Comment (cameron-eyds) [inline ppr-ui/src/components/common/RegistrationsWrapper.vue]: Small lint item

**Review by cameron-eyds — APPROVED** [review]
Looks good!

File: ppr-ui/src/components/common/RegistrationsWrapper.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/common/RegistrationsWrapper.vue]: Fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1907-skxdqt7v.web.app

**Review by doug-lovett — APPROVED** [review]
Very nice.

**Pull request merged**